### PR TITLE
⚡️ perf: Optimize Cache-Control

### PR DESCRIPTION
**Pull request by RubuJam**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [x] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 🔨 chore
- [x] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change
- 🐛  升级`@lobehub/ui`，由`^1.141.2`至`^1.142.4`，修复[💄 style: Expand Clear tooltip maxWidth](https://github.com/lobehub/lobe-chat/pull/2822)中，由于`@lobehub/ui`原有版本较低所产生的问题

- 💄 优化图片、`manifest.json`及`Open Graph`中的缓存控制
--- 
- 🐛 It is recommended that the `@lobehub/ui` package be upgraded from version `1.141.2` to `1.142.4`. This will address the issue identified in [💄 style: Expand Clear tooltip maxWidth](https://github.com/lobehub/lobe-chat/pull/2822), which was caused by the lower original version of `@lobehub/ui`.

- 💄 Optimize the cache control of images, `Open Graph` and `manifest.json`.

#### 📝 补充信息 | Additional Information
- 为`manifest.json`中的图片添加版本信息，若所涉图片更新，手动更改其版本号即可

- 为`Open Graph`(`src/app/metadata.ts` 和 `src/server/metadata.ts` )中的图片添加版本信息，若所涉图片更新，手动更改其版本号即可
--- 
- Add version information be added to the images included in the `manifest.json` file. In the event that the images in question undergo an update, it is sufficient to manually alter their version numbers.

- Add version information be added to the images included in the `Open Graph`( `src/app/metadata.ts` and `src/server/metadata.ts` ). In the event that the images in question undergo an update, it is sufficient to manually alter their version numbers.

--- 

Before：
Before

After：
After

### CODE REVIEW

File: next.config.mjs [hunk 1]
```diff
@@ -62,6 +62,47 @@ const nextConfig = {
...
+        source: '/screenshots/(.*).(png|jpe?g|gif|svg|ico|webp)',
+        headers: [
+          {
+            key: 'Cache-Control',
+            value: 'public, max-age=31536000, immutable',
+          },
+        ],
+      },
+      {
+        source: '/og/(.*).(png|jpe?g|gif|svg|ico|webp)',
```
Comment (arvinxx) [inline next.config.mjs:108]: og 不建议设成 31536000 ，会存在动态变化的场景的

Comment (RubuJam) [reply next.config.mjs:108]: 我直接连带`OG`里的照片和`icon`也加上版本信息，这样控制会更精细

**Comment by arvinxx** [comment]
@RubuJam 构建挂了

**Comment by RubuJam** [comment]
已经修复 @arvinxx 

already fixed @arvinxx

**Review by arvinxx — APPROVED** [review]
LGTM now, thanks for your contribution~

**Pull request merged**